integration-tests/vault-agroal/src/test/java/io/quarkus/vault/AgroalVaultITCase.java

```diff
@@ -19,6 +17,7 @@
 import io.quarkus.test.QuarkusUnitTest;
 import io.quarkus.test.common.QuarkusTestResource;
 import io.quarkus.vault.test.VaultTestLifecycleManager;
+import jakarta.inject.Inject;
```

Comment:
You should apply this PR first to avoid having all the imports moved: https://github.com/quarkiverse/quarkus-vault/pull/90 .

You can rebase and trigger a reformat with `mvn process-sources`. That will do the trick.